### PR TITLE
Upversion to 0.3.0 and update README to describe (wrap-routes ...)

### PR DESCRIPTION
**Pull request by scottyw**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(defproject puppetlabs/comidi "0.2.0-SNAPSHOT"
+(defproject puppetlabs/comidi "0.3.0-SNAPSHOT"
```
Comment (rlinehan) [inline project.clj]: Is this so the next release will be 0.3.0?

**Comment by rlinehan** [comment]
:+1: but can't be merged - needs a rebase/to be based off current HEAD of puppetlabs/master.

File: project.clj (hunk 1 above)
Comment (rlinehan) [inline project.clj]: I should've just read the PR title (facepalm)

**Comment by scottyw** [comment]
@rlinehan thanks! rebased.

**Comment by scottyw** [comment]
@dmcpl Nice spot - fixed

**Comment by rlinehan** [comment]
:+1:

**Pull request merged**